<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatDialog</class>
 <widget class="QWidget" name="ChatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>456</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>456</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>270</width>
       <height>40</height>
      </size>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QToolButton" name="tbVoice">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="ico.qrc">
        <normaloff>:/:/ico/voice.png</normaloff>:/:/ico/voice.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="tbVideo">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="ico.qrc">
        <normaloff>:/:/ico/video.png</normaloff>:/:/ico/video.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="tbNetHelp">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="ico.qrc">
        <normaloff>:/:/ico/help.png</normaloff>:/:/ico/help.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="tbShare">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="ico.qrc">
        <normaloff>:/:/ico/share.png</normaloff>:/:/ico/share.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="tbMedia">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="ico.qrc">
        <normaloff>:/:/ico/media.png</normaloff>:/:/ico/media.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTextEdit" name="showMsg">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFontComboBox" name="fontBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comBoxSize">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="currentText">
           <string>12</string>
          </property>
          <property name="currentIndex">
           <number>7</number>
          </property>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>17</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>18</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>21</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>22</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>23</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>24</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>26</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>36</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>42</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>72</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbB">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="accessibleName">
           <string/>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ico.qrc">
            <normaloff>:/:/ico/B.png</normaloff>:/:/ico/B.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbI">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="ico.qrc">
            <normaloff>:/:/ico/I.png</normaloff>:/:/ico/I.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbU">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="ico.qrc">
            <normaloff>:/:/ico/U.png</normaloff>:/:/ico/U.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbC">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="ico.qrc">
            <normaloff>:/:/ico/C.png</normaloff>:/:/ico/C.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTextEdit" name="sendMsg">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>110</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="close">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="send">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="friendInfo">
        <attribute name="title">
         <string>好友信息</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="other">
        <attribute name="title">
         <string>其他</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>发现文件：</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="userInfo">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>100</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ico.qrc"/>
 </resources>
 <connections/>
</ui>
